test/Altinn.AccessManagement.Tests/PolicyAdministrationPointTest.cs

```diff
@@ -31,7 +31,7 @@ public class PolicyAdministrationPointTest
         private readonly IDelegationChangeEventQueue _eventQueue;
         private readonly Mock<ILogger<IPolicyAdministrationPoint>> _logger;
         private DelegationMetadataRepositoryMock _delegationMetadataRepositoryMock;
-        private readonly ResourceRegistryClientMock _resourceRegistryClient;
+        private readonly ResourceRegistryClientMock _resourceRegistryClientMock = new ResourceRegistryClientMock();
 
```

Comment:
18 tests in this class related to deleting of policies are currently failing after merge. Looking into it.